roles/check_dependencies/tasks/main.yml

```diff
@@ -104,10 +104,21 @@
   register: internal_have_locate_cmd
   ignore_errors: yes
 
+- name: test locate command
```

Comment:
Let's add a `when: internal_have_locate_cmd`.  No need to test it when it is not there